docs/projected-volume.md

```diff
@@ -0,0 +1,42 @@
+# Projected Volumes and Windows Pods
+
+A [projected volume](https://kubernetes.io/docs/concepts/storage/volumes/#projected) allows mapping of several existing
+volume sources into the same directory in the container. With the [proposal for file permission handling in projected
+service account volume](https://github.com/kubernetes/enhancements/pull/1598) enhancement, the projected files have
+the correct permissions set including container user ownership for Linux pods when `RunAsUser` is set in the Pod 
+`SecurityContext`. However, for Windows pods this is not the case when `RunAsUsername` is set, due to differences in 
+the way user accounts are managed in Windows. Windows stores and manages local user and group accounts in a database
+file called Security Account Manager (SAM). This database is not shared between the Windows container host and the
+running containers. This prevents the kubelet from setting correct ownership on the files in the projected volume.
+This problem is tracked by [Windows Pod with RunAsUserName and a Projected Volume does not honor file
+permissions in the volume](https://github.com/kubernetes/kubernetes/issues/102849) issue.
+
+By default, the projected files will have the following ownership as shown for an example projected volume file:
+```powershell
+Path   : Microsoft.PowerShell.Core\FileSystem::C:\var\run\secrets\kubernetes.io\serviceaccount\..2021_08_31_22_22_18.318230061\ca.crt
+Owner  : BUILTIN\Administrators
+Group  : NT AUTHORITY\SYSTEM
+Access : NT AUTHORITY\SYSTEM Allow  FullControl
+         BUILTIN\Administrators Allow  FullControl
+         BUILTIN\Users Allow  ReadAndExecute, Synchronize
+Audit  :
+Sddl   : O:BAG:SYD:AI(A;ID;FA;;;SY)(A;ID;FA;;;BA)(A;ID;0x1200a9;;;BU)
+```
+This implies all administrator users like `ContainerAdministrator` will have read, write and execute access while,
+non-administrator users will have read and execute access.
```

Comment:
does this mean that one container could (if it were able to mount it), read the content of another serviceaccount's secret?

Reply:
Correct. I.e. if `HostPath` is used to give the Pod access to `c:\var\lib\kubelet\pods\`.